### ISSUE TITLE
Post-install message via `bridgetown new` has vanished

### ISSUE THREAD

**Issue by jaredcwhite** [bridgetownrb/bridgetown#327]
**Bridgetown Version**:
0-21-beta branch

The "Success!" message just isn't there. You see bundle and yarn install messages and then you get back to the shell prompt. Weird. This is definitely a blocker prior to the release of 0.21.

**Comment by jaredcwhite** [comment]
So this line in `Bridgetown::Commands::New#create_site` is where the logger gets broken. `logger.info` works before it, doesn't work after it.

```ruby
invoke(Webpack, ["setup"], {})
```

@ayushn21 I think you mentioned you were having an issue with testing and getting proper log output? Maybe it wasn't a test problem but an actual bug?!

**Comment by ayushn21** [comment]
Ahhh, I could have sworn it was working in the terminal and failing only in the tests. I've just looked again and I wasn't looking closely enough. We're doing `bundle install` and `yarn install` after invoking `Webpack`

```
 @skipped_bundle = true # is set to false if bundle install worked
bundle_install path unless options["skip-bundle"]

@skipped_yarn = true
yarn_install path unless options["skip-yarn"]
```

So I just saw that stuff was logging out and didn't look closely enough to see our success messages weren't being logged. Sorry, my bad!

I just ran `bridgetown new` with the `-c` flag and that doesn't log out our success message either. It seems the logger breaks whenever we call `invoke`. No clue why that might be though.

I can take a peek at it tomorrow at some point.

**Comment by ayushn21** [comment]
> I also just ran bridgetown new with the -c flag and that doesn't log out our success message either. It seems the logger breaks whenever we call invoke. No clue why that might be though.

Sorry, this isn't the case. I ran that test well past midnight last night so didn't remove the invocation of Webpack as brain as gone to sleep! 🤦 

Looks like invoking the webpack command is doing something funky to the logger; looking into it now

**Comment by ayushn21** [comment]
This is the offending line of code in `webpack.rb`:

```      
def site
  @site ||= Bridgetown::Site.new(configuration_with_overrides(quiet: true))
end
```

Any idea why creating an instance of the site might cause the logger to break? I can work around it for now, but I reckon it'll be good to understand why this happens. Maybe it was also the cause of the suddenly flaky test suite?

**Comment by jaredcwhite** [comment]
Fixed in #328, closing.
